src/fvsand_cuda.h

```diff
@@ -39,6 +39,20 @@ inline const char* gpuGetErrorString(gpuError_t err) { return cudaGetErrorString
 #define FVSAND_GPU_LAUNCH_FUNC(func, blocks, threads, sharedmem, stream, ...) \
     func<<<blocks, threads, sharedmem, stream>>>(__VA_ARGS__);
 
+constexpr int FVSAND_BLOCK_SIZE = 1024;
+#define FVSAND_GPU_KERNEL_LAUNCH(func, N, ... )                               \
+{                                                                             \
+  constexpr int SHARED_MEM  = 0;                                              \
+  constexpr int CUDA_STREAM = 0;                                              \
+  const int n_blocks = ( N + FVSAND_BLOCK_SIZE-1 ) / FVSAND_BLOCK_SIZE ;      \
```

Comment:
The intent with this macro is to encapsulate  the code for computing the number of cuda blocks inside the macro instead of repeating that at the call site where the kernel is launched.

Would this simplify how kernels are launched throughout the code?

This can be expanded upon to take in lambdas, or even write something like a `for_all` routine to do this. We could consider doing this as part of the hackathon. If nothing else, we could get more appreciation of what programming models like `RAJA` and `Kokkos` work underneath.